### PR TITLE
[8.0] fix: removed unused SubmissionMode CE parameter

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
FIX: removed unused SubmissionMode CE parameter

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/Resources/computingelements.rst [hunk 1]
```diff
@@ -117,17 +112,15 @@ automatically create the equivalent single core queues, see the :mod:`~DIRAC.Con
...
 
 A commented example follows::
 
    # Section placed in the */Resources/Sites/<domain>/<site>/CEs* directory
    ce01.infn.it
    {
-     CEType = CREAM
-     SubmissionMode = direct
-
+     CEType = HTCondor
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Resources/computingelements.rst]:
```suggestion
     CEType = HTCondorCE
```

File: docs/source/AdministratorGuide/Resources/computingelements.rst [hunk 2]
```diff
@@ -73,12 +73,7 @@ of the *ComputingElement* is located in the inside the corresponding site sectio
             ce01.infn.it
             {
               # Type of the CE
-              CEType = CREAM
-
-              # Submission mode should be "direct" in order to work with SiteDirector
-              # Otherwise the CE will be eligible for the use with third party broker, e.g.
-              # gLite WMS
-              SubmissionMode = direct
+              CEType = HTCondor
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Resources/computingelements.rst]:
```suggestion
              CEType = HTCondorCE
```

File: src/DIRAC/ConfigurationSystem/Client/Utilities.py [hunk 3]
```diff
@@ -266,8 +265,6 @@ def makeNewQueueName(queueName, ceType):
                 addToChangeSet((ceSection, "SI00", si00, newsi00), changeSet)
                 addToChangeSet((ceSection, "CEType", ceType, newCEType), changeSet)
                 addToChangeSet((ceSection, "MaxRAM", ram, newRAM), changeSet)
-                if submissionMode == "Unknown" and newSubmissionMode:
```
Comment (andresailer) [inline src/DIRAC/ConfigurationSystem/Client/Utilities.py:269]: also remove those lines referencing newSubmissionMode, like 255-256 above

Comment (fstagni) [reply src/DIRAC/ConfigurationSystem/Client/Utilities.py:269]: Done

**Review by andresailer — APPROVED** [review]

**Pull request merged**